x-pack/legacy/plugins/siem/public/components/source_destination/country_flag.tsx

```diff
@@ -13,7 +13,7 @@ import { pure } from 'recompose';
  * Example: `US` -> 🇺🇸
  */
 export const getFlag = (countryCode: string): string | null =>
-  countryCode.length === 2
+  countryCode && countryCode.length === 2
```

Comment:
Oh nice!

I think this fixes:
https://github.com/elastic/siem-team/issues/383

Feel free to close that ticket out once this goes across the line.

Reply:
oh totally forgot i even did this, meant to call it out when i put up the PR. Thanks!